packages/identity/stats.html

```diff
@@ -0,0 +1,3583 @@
+<!DOCTYPE html>
```

Comment:
Is this file intentionally added?

Reply:
oh, that was an artifact of some size analysis testing I was doing. I'll remove it